### PR TITLE
Cvu calc report

### PR DESCRIPTION
**Pull request by lmd59**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/product_report_test.rb [hunk 1]
```diff
@@ -221,6 +221,43 @@ class ProductReportTest < ActionController::TestCase
     end
   end
 
+  test 'should generate calculations in a report' do
+    user = User.create(email: 'vendor@test.com', password: 'TestTest!', password_confirmation: 'TestTest!', terms_and_conditions: '1')
```
Comment (lmd59) [inline test/unit/product_report_test.rb:225]: Can we just use the existing @user here?

File: test/unit/product_report_test.rb [hunk 2]
```diff
@@ -221,6 +221,43 @@ class ProductReportTest < ActionController::TestCase
...
+    vendor = FactoryBot.create(:vendor)
+    bundle = FactoryBot.create(:static_bundle)
+
+    measure_ids = ['BE65090C-EB1F-11E7-8C3F-9A214CF093AE', '40280382-5FA6-FE85-0160-0918E74D2075']
+    product = vendor.products.create(name: "my product #{rand}", cvuplus: true, randomize_patients: true, duplicate_patients: true,
+                                     bundle_id: bundle.id)
+
+    params = { measure_ids: measure_ids, 'cvuplus' => 'true' }
```
Comment (lmd59) [inline test/unit/product_report_test.rb:233]: I don't think we need cvuplus true here since it's in the product created above
